index.html

```diff
@@ -174,5 +175,37 @@ <h1>I'm Shimels, Glad to see you!</h1>
         </li>
       </ul>
     </section>
+    <section id="contact-form">
+      <h1>Contact me</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form action="https://formspree.io/f/xyyqzzjg" method="post">
+        <input
+          type="text"
+          id="name"
+          name="user_name"
+          placeholder="Write your name here"
+          maxlength="30"
+          required
+        />
+        <input
+          type="email"
+          id="email"
+          name="user_email"
+          placeholder="Write your email here"
+          required
+        />
+        <textarea
+          name="user_message"
+          placeholder="Write your message here"
+          maxlength="500"
+          required
+        ></textarea>
+        <button type="submit" class="content-submit-btn">Get in touch</button>
+      </form>
+    </section>
```

Comment:
👍📱 Excellent work, your code appears almost identical on the 375px screen size! However, there are some improvements needed to meet project requirements for mobile responsiveness:

- [ ] The "Contact Us" section needs to be responsive for mobile devices of different screen sizes. It should work well from a minimum mobile screen size of 375px, and errors in screen sizes less than 375px can be ignored. Furthermore, ensure the mobile version remains responsive until the desktop screen size of 768px.

[Portfolio.webm](https://github.com/shimels1/portfolio_microverse/assets/55840999/58d61327-7276-4d96-bcbf-512c29e3d209)

- [ ] Instead of using fixed width and height for most elements, it is recommended to utilize responsive values in your CSS rules, such as percentages (e.g., width: 80%), rather than pixels. This will ensure that your layout adapts smoothly to various screen sizes.
![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/8b7488e3-b3f1-46ed-8524-a6ccf168343e)
- [ ] Additionally, I noticed that the child tags of the section have separate margins, and the margin appears to be excessive compared to the design. To achieve a more professional look, consider using the gap property and setting a proper gap size in accordance with the design.
- [ ] Moreover, you have a padding of 20px in the left-right side. But as per design, it should be 24px. please follow the design and update your code to match it. 🙏 
![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/0f5c8962-9fe1-49b0-b97e-d33d42e2c9e2)

By addressing these points, your "Contact Us" section will be more responsive and visually aligned with the project requirements. Keep up the great work, and I look forward to seeing the improvements! 🚀📱🎨